Parse/src/main/java/com/parse/Parse.java

```diff
@@ -158,72 +152,73 @@ public Builder server(String server) {
       }
 
       /**
-       * Add a {@link ParseNetworkInterceptor}.
-       *
-       * @param interceptor The interceptor to add.
-       * @return The same builder, for easy chaining.
-       */
-      public Builder addNetworkInterceptor(ParseNetworkInterceptor interceptor) {
-        if (interceptors == null) {
-          interceptors = new ArrayList<>();
-        }
-        interceptors.add(interceptor);
-        return this;
-      }
-
-      /**
        * Enable pinning in your application. This must be called before your application can use
        * pinning.
+       *
        * @return The same builder, for easy chaining.
        */
       public Builder enableLocalDataStore() {
         localDataStoreEnabled = true;
         return this;
       }
 
-      /* package for tests */ Builder setNetworkInterceptors(Collection<ParseNetworkInterceptor> interceptors) {
```

Comment:
I believe https://github.com/parse-community/ParseInterceptors-Android is redundant.  Before we upgraded the library to rely mostly on the OkHttp networking, this repo was used to provide a more generic interface for the different ways to handle networking.  

Most of the Android world relies on Stetho/HttpLogInterceptor that provides out of the box so I don't see a reason we need to continue maintaining ParseInterceptors (also it is not really maintained)